lib/puppet/property/list.rb

```diff
@@ -7,19 +7,6 @@ class Property
     #
     class List < Property
 
-      def should_to_s(should_value)
-        #just return the should value
-        should_value
-      end
-
-      def is_to_s(currentvalue)
-        if currentvalue == :absent
-          return _("absent")
-        else
-          return currentvalue.join(delimiter)
```

Comment:
I don't think absent should be translated, as that value has special meaning, as it corresponds to one value in the `ensurable` enumeration.